### DISCUSSION TITLE
How to install on Production Server

### DISCUSSION THREAD

**Discussion by xmateosx** [quartznet/quartznet#2166] [General]
I have published Quartz.net to a Production machine folder: Default Web Site\Quartz, and manually ran the exe. I'm building off of this part of the code: "Quartz.Examples.AspNetCore.exe" 
Can you point me to documentation about how manage deployments better? This is running on an intranet.
And how do you check to see if it stopped running and start it again?
And finally, I can access the Home Page http://localhost:5000/ from the Production Server, but I can't access it from my Dev Machine:
http://prodserver:5000
Thanks!

**Comment by lahma** [comment]
I'm afraid this has very little to do with Quartz itself. You should examine ASP.NET hosting documentation available from Microsoft and how to expose servers and how to maintain them.

**Reply by xmateosx** [reply]
Just to add to this, 

While on the Production Machine I can access the Home Page at http://localhost:5000/ 
While on the Production Machine I cannot access the Hope Page from http://prodserver:5000/ 

Am I the only person struggling with this?
I have other websites that are running just fine. 
Can somebody post their web.config file?

Here is mine:
```

<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <location path="." inheritInChildApplications="false">
    <system.webServer>
      <handlers>
        <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModuleV2" resourceType="Unspecified" />
      </handlers>
      <aspNetCore processPath="dotnet" arguments=".\Quartz.Examples.AspNetCore.dll" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" hostingModel="inprocess" />
    </system.webServer>
  </location>
</configuration>
<!--ProjectGuid: 288f72cb-4559-45ed-9199-f4594bed026b-->``

```
